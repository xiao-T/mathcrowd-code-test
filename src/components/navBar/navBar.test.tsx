/**
 * NavBar component test
 * Thu Jul 23 10:39:11 2020
 * by xiaoT
 */
import React, { FC } from 'react'
import ReactDom from 'react-dom'
import { shallow, mount } from 'enzyme'
import { act } from 'react-dom/test-utils'
import { getByLabelText, fireEvent } from '@testing-library/react'

import mountWithProvider from '@Store/mockStore'
import NavBar from './'

describe('NavBar Test', () => {
  
})

