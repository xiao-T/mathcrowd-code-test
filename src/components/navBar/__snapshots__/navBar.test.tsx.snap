// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Test should be match the snapshot 1`] = `"<div class=\\"nav-bar\\"><div data-justify=\\"flex-end\\" data-align=\\"center\\" class=\\"nav-bar-inner flex\\"><form><label data-align=\\"center\\" class=\\"form-item flex\\"><i class=\\"icon icon-search\\"></i><input name=\\"searchKey\\" class=\\"flex-1\\" placeholder=\\"请输入关键字\\" type=\\"text\\" value=\\"\\"></label></form><i class=\\"icon icon-home\\"></i><i class=\\"icon icon-bell\\"></i><i class=\\"icon icon-avatar\\"></i></div></div>"`;
